src/_pytest/config/argparsing.py

```diff
@@ -127,19 +128,24 @@ def parse_known_and_unknown_args(
         args = [str(x) if isinstance(x, py.path.local) else x for x in args]
         return optparser.parse_known_args(args, namespace=namespace)
 
-    def addini(self, name, help, type=None, default=None):
+    def addini(self, name, help, type=None, default=None, choices=None):
```

Comment:
Let's take this opportunity and make `choices` keyword-only

```suggestion
    def addini(self, name, help, type=None, default=None, *, choices=None):
```